modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -408,6 +409,14 @@ public ObjectDefinition publishCustomObjectDefinition(
 			throw new ObjectDefinitionStatusException();
 		}
 
+		int objectFieldsAmount =
+			_objectFieldPersistence.countByObjectDefinitionId(
+				objectDefinition.getObjectDefinitionId());
+
+		if (objectFieldsAmount == 0) {
+			throw new RequiredObjectFieldException();
+		}
+
```

Comment:
@p-albuquerque can you suppress this variable and put it inside the if condition? Like this:

```
if (_objectFieldPersistence.countByObjectDefinitionId(
            objectDefinition.getObjectDefinitionId()) == 0) {
			throw new RequiredObjectFieldException();
		}
```

Reply:
I tried this, however the FormatSource failed to build, with the follow formatting issue:
"Create a new variable for the left hand side operand of the '==' operator for better readability"

Reply:
I've tried this, however the FormatSource failed to build, with the follow formatting issue:
"Create a new variable for the left hand side operand of the '==' operator for better readability"

Reply:
Ok! Thx, Paulo!

Reply:
Can you rename it for just `count` ?

Reply:
Done.